config/gunicorn/dev.py

```diff
@@ -0,0 +1,20 @@
+"""Gunicorn *development* config file"""
+
+# Django WSGI application path in pattern MODULE_NAME:VARIABLE_NAME
+wsgi_app = "wsgi:application"
+# The granularity of Error log outputs
+loglevel = "debug"
+# The number of worker processes for handling requests
+workers = 2
+# The socket to bind
+bind = "0.0.0.0:8000"
+# Restart workers when code changes (development only!)
+reload = False
+# Write access and error info to /var/log
+# accesslog = errorlog = "/var/log/gunicorn/dev.log"
+# Redirect stdout/stderr to log file
+capture_output = True
+# PID file so you can easily fetch process ID
+pidfile = "/var/run/gunicorn/dev.pid"
+# Daemonize the Gunicorn process (detach & enter background)
+daemon = False
```

Comment:
Do we foresee having a separate wsgi file for each environment? How are we going to switch between files in the gunicorn command? Should we roadmap out a task to parameterize this file and pull the values from an `.env` file?

Reply:
We could just change the CMD for each container in ECS to include the relevant parameters in the command. Most of these are probably not necessary. Can't see any reason why there would be differences between environments, but then again I don't have too much gunicorn experience.